addons/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/handler/IhcHandler.java

```diff
@@ -413,35 +419,41 @@ private void sendPulseCommand(ChannelUID channelUID, ChannelParams params, WSRes
         Converter<WSResourceValue, Type> converter = ConverterFactory.getInstance().getConverter(value.getClass(),
                 OnOffType.class);
-        ConverterAdditionalInfo converterAdditionalInfo = new ConverterAdditionalInfo(null, params.isInverted());
 
-        WSResourceValue valOn = converter.convertFromOHType(OnOffType.ON, value, converterAdditionalInfo);
-        WSResourceValue valOff = converter.convertFromOHType(OnOffType.OFF, value, converterAdditionalInfo);
+        if (converter != null) {
+            ConverterAdditionalInfo converterAdditionalInfo = new ConverterAdditionalInfo(null, params.isInverted(),
+                    getCommandLevels(params));
 
-        // set resource to ON
-        logger.debug("Update resource value (inverted output={}): {}", params.isInverted(), valOn);
-        if (updateResource(valOn)) {
-            scheduler.submit(() -> {
-                // sleep a while
-                try {
-                    logger.debug("Sleeping: {}ms", pulseWidth);
-                    Thread.sleep(pulseWidth);
-                } catch (InterruptedException e) {
-                    // do nothing
-                }
-                // set resource back to OFF
+            WSResourceValue valOn = converter.convertFromOHType(OnOffType.ON, value, converterAdditionalInfo);
+            WSResourceValue valOff = converter.convertFromOHType(OnOffType.OFF, value, converterAdditionalInfo);
 
-                logger.debug("Update resource value (inverted output={}): {}", params.isInverted(), valOff);
-                try {
-                    if (!updateResource(valOff)) {
-                        logger.warn("Channel {} update to resource '{}' failed.", channelUID, valOff);
+            // set resource to ON
+            logger.debug("Update resource value (inverted output={}): {}", params.isInverted(), valOn);
+            if (updateResource(valOn)) {
+                scheduler.submit(() -> {
+                    // sleep a while
+                    try {
+                        logger.debug("Sleeping: {}ms", pulseWidth);
+                        Thread.sleep(pulseWidth);
+                    } catch (InterruptedException e) {
+                        // do nothing
                     }
-                } catch (IhcExecption e) {
-                    logger.error("Can't update channel '{}' value, cause ", channelUID, e.getMessage(), e);
-                }
-            });
+                    // set resource back to OFF
+
+                    logger.debug("Update resource value (inverted output={}): {}", params.isInverted(), valOff);
+                    try {
+                        if (!updateResource(valOff)) {
+                            logger.warn("Channel {} update to resource '{}' failed.", channelUID, valOff);
+                        }
+                    } catch (IhcExecption e) {
+                        logger.error("Can't update channel '{}' value, cause ", channelUID, e.getMessage(), e);
```

Comment:
I know this was already there before. But this is a good moment to change this logging to error to a warn, because logging to error is reserved in openHAB for system wide failures.
Also there are 2 parameters an only 1 in the format string:
```suggestion
                        logger.error("Can't update channel '{}' value, cause: {}", channelUID, e.getMessage(), e);
```